unit_test/test_spdm_crypt/test_spdm_crypt.c

```diff
@@ -401,6 +505,29 @@ void libspdm_test_crypt_spdm_x509_certificate_check(void **state)
         assert_true(status);
         free(file_buffer);
     }
+
+    uint8_t buffer[0x1200];
+    size_t buffer_size;
+    uint8_t data;
+
+    if ((LIBSPDM_RSA_SSA_2048_SUPPORT) && (LIBSPDM_SHA256_SUPPORT)) {
+        libspdm_copy_mem(buffer, m_cert_buffer_size, m_cert_buffer, m_cert_buffer_size);
+        buffer_size = m_cert_buffer_size;
+
```

Comment:
We can adding one normal case to comparing with the following non canonical encoding format case.
`status = libspdm_x509_certificate_check(buffer, buffer_size,`